pkg/auth/auth_policy.go

```diff
@@ -36,6 +36,8 @@ type Policy interface {
 	// returned.
 	AuthorizeApp(manifest Manifest, logger logging.Logger) error
 
+	Authorize(email, appUser string) bool
```

Comment:
is this an email? At Square, I bet this is an LDAP handle. How about calling this "username" or "handle" instead

Reply:
It will be an email in the form of ldap + "@squareup.com", if that is considered an username then I am up for renaming this.

Reply:
it is an email because that's how PGP works

Reply:
but in practice we generate the email by adding a username and a domain

Reply:
wild. OK, good enough.